modules/apps/27-interchain-accounts/controller/ibc_middleware_test.go

```diff
@@ -185,6 +186,11 @@ func (suite *InterchainAccountsTestSuite) TestOnChanOpenInit() {
 				isNilApp = true
 			}, true,
 		},
+		{
+			"middleware disabled", func() {
+				suite.chainA.GetSimApp().ICAControllerKeeper.DeleteMiddlewareEnabled(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
```

Comment:
I think in this case, we should also set the IBC mock callback to return an error if called

Reply:
ayy lmao! was going to but laziness prevailed, I'll push an update :)